src/js/components/Select/README.md

```diff
@@ -131,10 +131,28 @@ boolean
 **disabled**
 
 Whether the entire select or individual options should be disabled.
+        When an array is provided, it indicates the indexes into 'options' of
+        the disabled options. When a function is provided, it is called with
+        each option to determine if the option should be disabled.
 
 ```
 boolean
 [number]
+function
+```
+
+**disabledKey**
```

Comment:
Could this also be an array of disabledKeys? That would be super convenient and parallel the `value` array possibility.

Reply:
Good idea. I've added this as part of `disabled` though. So, `disabled` parallels `value` and `disabledKey` parallel `valueKey`.